firestore/templates/dashboard.html

```diff
@@ -20,17 +20,18 @@ <h3 id="guesses"></h3>
     <script src="https://www.gstatic.com/firebasejs/4.9.1/firebase-firestore.js"></script>
     <script>
       // Firebase Configuration - REPLACE WITH YOUR OWN VALUES
-      var config = {
-        apiKey: "REPLACE_WITH_YOUR_API_KEY",
-        authDomain: "REPLACE_WITH_YOUR_AUTH_DOMAIN",
-        databaseURL: "REPLACE_WITH_YOUR_DATABASE_URL",
-        projectId: "REPLACE_WITH_YOUR_PROJECT_ID",
-        storageBucket: "REPLACE_WITH_YOUR_STORAGE_BUCKET",
-        messagingSenderId: "REPLACE_WITH_YOUR_MESSAGING_SENDER_ID"
+      var firebaseConfig = {
```

Comment:
let's remove these actual values and change back to the "REPLACE_WITH_YOUR_ ... " placeholders

Reply:
Fixed.
`appId` comes with the copy/paste, so I included it.